wren-ui/src/apollo/server/repositories/threadResponseRepository.ts

```diff
@@ -70,10 +70,18 @@ export class ThreadResponseRepository
       })
       .map((res) => {
         // JSON.parse detail and error
+        const detail =
+          res.detail && typeof res.detail === 'string'
```

Comment:
The return data type from pg is object but the data type from sqlite is string.
We only need to parse the value that comes from SQLite.